XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/BaseControllerDataProvider.cs

```diff
@@ -45,7 +45,7 @@ protected virtual IMixedRealityPointer[] RequestPointers(SystemType controllerTy
                     if (((useSpecificType && pointerProfile.ControllerType.Type == controllerType.Type) || (!useSpecificType && pointerProfile.ControllerType.Type == null)) &&
                         (pointerProfile.Handedness == Handedness.Any || pointerProfile.Handedness == Handedness.Both || pointerProfile.Handedness == controllingHand))
                     {
-                        var pointerObject = Object.Instantiate(pointerProfile.PointerPrefab, MixedRealityToolkit.CameraSystem?.CameraRig.PlayspaceTransform);
+                        var pointerObject = Object.Instantiate(pointerProfile.PointerPrefab, MixedRealityToolkit.CameraSystem?.MainCameraRig.PlayspaceTransform);
```

Comment:
We should check if the camera system is null here and fall back to `CameraCache.Main.parent`